examples/multiphysics/laser_melting/laser_melting.prm

```diff
@@ -0,0 +1,200 @@
+	set end time 			= 0.0034
+	set beam orientation 		= y
+	subsection path
+		set Function expression =  0.5 * t; 0.000500
+	end
+end
+
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+subsection initial conditions
+    set type = nodal
+    subsection uvwp
+        set Function expression 	= 0; 0; 0
+    end
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+  set number of fluids     		= 1
+  subsection fluid 0
+    set thermal conductivity model 	= phase_change
+    set rheological model 		= phase_change
+    set specific heat model 		= phase_change
+    
+    set density 			= 4300
+    
+    subsection phase change
+      # Enthalpy of the phase change
+      set latent enthalpy      		= 286000
+
+      # Temperature of the liquidus
+      set liquidus temperature 		= 1750
+
+      # Temperature of the solidus
+      set solidus temperature  		= 1700
+
+      # Specific heat of the liquid phase
+      set specific heat liquid 		= 830
+
+      # Specific heat of the solid phase
+      set specific heat solid  		= 200
+      
+      set thermal conductivity solid 	= 1.5
+      set thermal conductivity liquid 	= 33.4
+
+      # viscosity of the liquid phase
+      set viscosity liquid 		= 6
+
+      # viscosity of the solid phase
+      set viscosity solid  		= 100000
+    end
+
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+        set type = dealii
+        set grid type = subdivided_hyper_rectangle
+        set grid arguments 	= 8, 2 : -0.0001, 0 : 0.0019, 0.000500 : true
+        set initial refinement 	= 8
+end
+
+#---------------------------------------------------
+# Mesh Adaptation
+#---------------------------------------------------
+subsection mesh adaptation
+  set type                    = kelly
+  set variable                = temperature
+  set fraction type           = fraction
+  set max refinement level    = 8
+  set min refinement level    = 4
+  set frequency               = 1
+  set fraction refinement     = 0.5
+  set fraction coarsening     = 0.2
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 4
+    subsection bc 0
+        set id = 0
+        set type              = noslip
+    end
+    subsection bc 1
+        set id = 1
+        set type              = noslip
+    end
+    subsection bc 2
+        set id = 2
+        set type              = noslip
+    end
+    subsection bc 3
+        set id = 3
+        set type              = noslip
+    end
+end
+subsection boundary conditions heat transfer
+  set number                  = 4
+    subsection bc 0
```

Comment:
Since all BCs have the same actual BC, why not use a single boundary condition by setting colorize=false in the mesh creation?

Reply:
I think it's better to keep it because the user is able to easily define different boundary conditions by modifying the parameters. For instance, since the temperature of the walls are not equal after applying the laser heat source, maybe the users would be interested in defining different emissivity or ambient temperatures for the boundaries.